sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -594,9 +593,7 @@ export { WebSocketImpl }
 export { WebSocketOptions }
 
 // @public
-export type WithResponse<T extends object> = T & {
-    _response: HttpResponse;
```

Comment:
We should retain this _response in the output

Reply:
Yes, @sarangan12 has been investigating the compatibility of HttpResponse and PipelineResponse. Will re-visit this.

Reply:
@sarangan12 My investigation is in progress. After the last meeting, I am meeting with Jeff on 1/4 to discuss this further. Will update everyone after the meeting

Reply:
@sarangan12's core-http-compatibility doesn't have mapping of responses yet so I copied some code from core-http to service bus and add transformation.  Looks that in SB we only needed `request`, `status`, and `headers` of the response, no body so it is simpler here.